reliability/stub-core/src/main/kotlin/com/datadog/android/core/stub/StubInternalLogger.kt

```diff
@@ -12,6 +12,8 @@ import com.datadog.android.core.metrics.TelemetryMetricType
 
 @Suppress("UnsafeThirdPartyFunctionCall")
 internal class StubInternalLogger : InternalLogger {
+
+    var telemetryEventsWritten = mutableListOf<Map<String, Any>>()
```

Comment:
no need to be mutable reference.

```suggestion
    val telemetryEventsWritten = mutableListOf<Map<String, Any>>()
```

Reply:
fixed